core/src/types/operator/operator.rs

```diff
@@ -1254,6 +1254,25 @@ impl Operator {
     /// # }
     /// ```
     ///
+    /// ## `if_not_exist`
+    ///
+    /// Set `if_not_exist` for this `write` request. This can be treated as a simplified version
```

Comment:
Hi, `if_not_exist` is not the same feature as `if_none_match`. For example, S3 only supports `if_not_exist` but not `if_none_match`. We should eliminate use cases like `if_none_match("*")` to clarify this.

All existing tests and docs should replace by `if_not_exist` instead.

Reply:
I'm not familiar with the current usage of `if_none_match`. I can find it in op read/stat/write.

1. Should we keep the tests since we still allow users to use this option?
2. Do you mean that we should delete the `if_none_match` from the write-op-related docs?

Reply:
> 1. Should we keep the tests since we still allow users to use this option?

The tests using `if_none_match("*")` should be removed since we can't guarantee that all services will support this.

> 2. Do you mean that we should delete the `if_none_match` from the write-op-related docs?

We can retain this docs, but we need to remove the `if_none_match("*")` example. Instead, we should use an `etag` to clarify that we are accepting an `etag`.